docs/src/guides/advanced/19_interop_basics.md

```diff
@@ -0,0 +1,197 @@
+# Interop basics
+
+This guide explains the basic process of using the InteropMessage interface from protocol version v29. With this
+interface, a message is sent from one chain (chain A) and then verified on another chain (chain B).
+
+Note: This interface only broadcasts messages. It does not include features such as specifying a destination or replay
+protection, which are available in higher-level interfaces.
+
+## Preparation
+
+### Setup chains
+
+Follow the instructions at: https://github.com/matter-labs/zksync-era/blob/main/docs/src/guides/launch.md
+
+This will help you set up the Era chain and the associated gateway. In this example, we will send and verify the message
+on Era. If you prefer to test on another chain (like era2), launch a new chain following the guide above and migrate it
+to the gateway.
+
+### Fund an Account
+
+Give funds to an account on your local Era using the following commands:
+
+```
+zkstack dev rich-account
+ACCOUNT="0x36615Cf349d7F6344891B1e7CA7C72883F5dc049"
+PRIVATE_KEY="0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110"
+```
+
+## Sending a Message
+
+While messages are usually sent by contracts, you can send one directly using:
+
+```
+cast send -r http://localhost:3050 0x0000000000000000000000000000000000008008 "sendToL1(bytes)" "0x1234" --private-key $PRIVATE_KEY
+```
+
+Here, `0x...8008` is the L1Messenger, a special system contract. Remember the transaction hash, as we'll use it in
+commands below.
+
+Next, retrieve the interop proof from the sequencer (replace the transaction hash with your own):
```

Comment:
It probably makes sense to mention that the proof might not be available right away (it took me a few iterations of method returning `null` until I got the proof).